StripeFinancialConnections/StripeFinancialConnections/Source/Native/PartnerAuth/PartnerAuthViewController.swift

```diff
@@ -20,6 +20,10 @@ protocol PartnerAuthViewControllerDelegate: AnyObject {
         _ viewController: PartnerAuthViewController,
         didCompleteWithAuthSession authSession: FinancialConnectionsAuthSession
     )
+    func partnerAuthViewController(
```

Comment:
should these three be documented on which needs to be used at which case?

Reply:
these are just standard delegate methods not accessible by anyone

you may be familiar, but these type of `Delegate`'s (/patterns) are quite common and documenting them would be akin to documenting every function written in a class (or, in other words, not much value / its redundant)